login.html

```diff
@@ -17,25 +17,36 @@
   <body>
     <div class="form_container">
       <a href="index.html">
-        <img
-          class="form_logo_img"
-          src="img/login_logo.png"
-          alt="로그인페이지_판다마켓_로고_이미지"
-        />
+        <img class="form_logo_img" src="img/login_logo.png" alt="판다마켓" />
       </a>
-      <p class="form_text_input">이메일</p>
-      <input type="text" class="form_input" />
-      <p class="form_text_password">비밀번호</p>
-      <input type="password" class="form_input_password" />
-      <button class="form_button">로그인</button>
+      <form class="form_login" onsubmit="login">
+        <label class="form_text_input" for="email">이메일</label>
+        <input
+          id="email"
+          type="text"
+          class="form_input"
+          placeholder="이메일을 입력해주세요"
+        />
+        <label class="form_text_password" for="password">비밀번호</label>
+        <div class="password_wrap">
+          <input
+            id="password"
+            type="password"
+            class="form_input_password"
+            placeholder="비밀번호를 입력해주세요"
+          />
+          <input type="button" class="password_eye" />
```

Comment:
❗️ 수정요청
button 태그를 쓰시는게 더 현대적인 방식일 것 같아요~
```suggestion
          <button type="button" class="password_eye" />
```